build.gradle.kts

```diff
@@ -1,8 +1,14 @@
+buildscript {
+    dependencies {
+        classpath("org.jetbrains.kotlin:kotlin-gradle-plugin:1.7.21")
+    }
+}
 @Suppress("DSL_SCOPE_VIOLATION")
 plugins {
     alias(libs.plugins.android.application) apply false
     alias(libs.plugins.android.library) apply false
     alias(libs.plugins.kotlin.android) apply false
+    alias(libs.plugins.ksp) apply false
```

Comment:
esse aqui precisa para que mesmo? é o kapt ne?

Reply:
nossa faz tempo que não configuro room, mas precisa desse ?

Reply:
Issoo. Atualmente, eles estão recomendando tirar o kapt do seu projeto para usar o ksp, pois ele [melhora a performance do build](https://kotlinlang.org/docs/ksp-why-ksp.html#comparison-to-kapt).

- [Migrate to KSP](https://developer.android.com/build/migrate-to-ksp)